### PR TITLE
Finding intersection of two linked lists in python 

### PR DESCRIPTION
**Pull request by Nikitha2309**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes, I have clearly read the Guidelines, and followed the given code template format, and read READ.ME file and linked it to the code file

### Description

Find the Intersection point of the given two linked lists. If they do not intersect, print " The given linked lists do not intersect".

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Fixes Issue #3833 

@tarunyadav148 I request you to review my PR and merge it. Thank you

### CODE REVIEW

**Comment by Nikitha2309** [comment]
Respected, Though the solution is made perfectly following all the given guidelines according to the code template format ,
Some of the minor silly bot checks like blank space in comments are making 2 of the bot checks to fail .

* The code works efficiently for all the corner test cases though. 
* It is understandable with comments 
* It takes in dynamic user-friendly inputs 
* It provides understandable outputs
* Most importantly it is the optimised solution 

Hence I hope those 2 bot checks won't matter 
I tried modifying some things according to the bot checks which were failing already
But I think some checks like blank spaces in comments etc, doesn't matter

@HarshCasper @anushkrishnav @tarunyadav148 @iamrajiv @ankitaggarwal23 
I request someone to review the code and merge it before any other conflicts arise ( because of other pull requests due to modifications in READ.ME file) 

Thank You

File: Python/cp/find_intersection_of_two_linked_lists.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+# Enter the values of each node of Linked List 2 in order : 5 4 3 4 5 6 7   
+
+# Output Format :
+# Example : ( Output to the above input example ) 
+# Linked List 1 = 1->2->3->4->5->6->7->None
+# Linked List 2 = 5->4->3->4->5->6->7->None
+# The intersection point is 3
+
+# Corner Test Cases Result
+# Code works fine for all corner test cases tested , including null linked lists.
```
Comment (HarshCasper) [inline Python/cp/find_intersection_of_two_linked_lists.py]: Put this in a multi-line string. Add a new line at the EOF

File: Python/cp/find_intersection_of_two_linked_lists.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+# Example :
+#     Input :
+#         Enter the values of each node of Linked List 1 in order : 1 2 3 4 5 6 7
+#         Enter the values of each node of Linked List 2 in order : 5 4 3 4 5 6 7 
+#     Output :
+#         Example : ( Output to the above input example ) 
+#         Linked List 1 = 1->2->3->4->5->6->7->None
+#         Linked List 2 = 5->4->3->4->5->6->7->None
+#         The intersection point is 3
+
```
Comment (HarshCasper) [inline Python/cp/find_intersection_of_two_linked_lists.py]: Remove this

File: Python/cp/find_intersection_of_two_linked_lists.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+# Purpose
+# To write the driver code that takes input and a 
+# function that finds the intersection of given two linked lists
+
+# Method
+# Calculate size of both given linked lists by counting no.of nodes 
+# and find difference between them , say d.
+# Now traverse the bigger list from the first node till d nodes 
+# so that from here onwards both the lists have equal no of nodes.
+# Then traverse both the lists in parallel till we come across a common node.
```
Comment (HarshCasper) [inline Python/cp/find_intersection_of_two_linked_lists.py]: Remove them as well. Keep method embedded in the  code

File: Python/cp/find_intersection_of_two_linked_lists.py [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        
+# Time/Space Complexity
+# Time Complexity : O(m+n)
+#   where m and n are size of two linked lists
+# Space Complexity : O(1)
+#   No additinal space is required
```
Comment (HarshCasper) [inline Python/cp/find_intersection_of_two_linked_lists.py]: Shift Time and Space Complexity at the end. Typo in `additinal`

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by Nikitha2309** [comment]
Thanks for the suggestions @HarshCasper , 
I have done all changes suggested 
Pls review it once again . I think it should be fine now

Thank you

**Review by HarshCasper — APPROVED** [review]

**Comment by Nikitha2309** [comment]
Thank you for the merge  :)
Looking forward to contribute more to this repo .

**Pull request merged**